src/vector_index.c

```diff
@@ -47,7 +47,19 @@ static VecSimIndex *openVectorKeysDict(RedisSearchCtx *ctx, RedisModuleString *k
   // create new vector data structure
   kdv = rm_calloc(1, sizeof(*kdv));
   kdv->p = VecSimIndex_New(&fieldSpec->vecSimParams);
-  dictAdd(ctx->spec->keysDict, keyName, kdv);
+  VecSimIndexInfo indexInfo = VecSimIndex_Info(kdv->p);
+  switch (indexInfo.algo)
+  {
+    case VecSimAlgo_BF:
+      spec->stats.invertedSize = indexInfo.bfInfo.memory;
+      break;
+    case VecSimAlgo_HNSWLIB:
+      spec->stats.invertedSize = indexInfo.hnswInfo.memory;
```

Comment:
```suggestion
      spec->stats.invertedSize += indexInfo.hnswInfo.memory;
```

Reply:
`spec->stats` is a for all fields.

Reply:
@MeirShpilraien @DvirDukhan  WDYT about adding `VectorSize` field to `IndexStats`?
This isn't an inverted index structure and might mislead users.